### ISSUE TITLE
Do you want to add the OCR module to this project?

### ISSUE THREAD

**Issue by walterpadua** [php-opencv/php-opencv#39]
Do you want to add the OCR module to this project?

**Comment by morozovsk** [comment]
Hi. You can find all available functions in that file. You can put it in your project for autocomplete.
OCR functions are not available. I've newer used it.
So if you need it you can try add it like similar functions/classes:
https://github.com/php-opencv/php-opencv/blob/master/source/opencv2/opencv_imgproc.cc
https://github.com/php-opencv/php-opencv/blob/master/source/opencv2/core/opencv_type.cc

**Comment by wpj16** [comment]
Yes, I'm already using these image processing functions, and it works very well, but something that would make this project in great demand, would be to support the native functions that opencv makes available when it is compiled with Tesseract, as it would bring together the best of 2 worlds in 1.

**Comment by scramatte** [comment]
+1 We are interested to use such feature for APNR (Plate detection)
We can sponsor the development, take a look to #46 ticket
